Fusion content (#7794)
fishtown-analytics/docs.getdbt.com: website/docs/docs/about-dbt-install.md

---
title: About dbt installation
id: about-dbt-install
description: "Learn how to install dbt in local environments"
sidebar_label: "About dbt installation"
pagination_next: "docs/core/dbt-core-environments"
pagination_prev: null
---

> <Constant name="dbt" />  enables data teams to transform data using analytics engineering best practices. You can install dbt locally using these options:

Reviewer: ```suggestion
<Constant name="dbt" /> enables data teams to transform data using analytics engineering best practices. You can install dbt locally using these options:
```

> * Local command line interface (CLI) tools leveraging the speed and scale of the dbt Fusion engine or using our legacy Core product

Reviewer: ```suggestion
* Local command line interface (CLI) tools leveraging the speed and scale of the <Constant name="fusion_engine" /> or using our legacy Core product
```

* VS Code and Cursor with dbt extension

> If you're interested in using dbt platform, our feature-rich, browser-based UI, you can learn more in [About dbt set up](/docs/cloud/about-cloud-setup).

Reviewer: ```suggestion
If you're interested in using the <Constant name="dbt_platform" />, our feature-rich, browser-based UI, you can learn more in [About dbt set up](/docs/cloud/about-cloud-setup).
```
